examples/installation_on_windows/README.md

```diff
@@ -0,0 +1,57 @@
+# Install SLLM on Windows Machines
+
+## 1 Windows Subsystem for Linux
+To install the SLLM dependencies, Ubuntu 20.04 is required by the installation guidance states. Packages, like vllm, cannot be installed on Windows. Therefore, the Windows machines require a sub-system to run Linux. Here, it's recommended downloading the Ubuntu app from Microsoft store and the WSL connection will allow to you install everything needed for. If you have more familiar to start Linux environment, please feel free to go ahead and share your experience with us. Generally speaking, it is a developed and stable app except WSL connection doesn't support GUI, and sometimes it can be nasty to set variable path.
+
+![alt text](Ubuntu-app.png)
```

Comment:
It would be preferable to provide another picture in a fully English version.

Reply:
fully English verision updated